Conversion-Class Temperatura

```diff
@@ -0,0 +1,14 @@
+package com.company;
+
+public class Temperatura {
+
+    public double temperatura;
+    public void setTemperatura(double temperaturaZilei) {
```

Comment:
you also need to put back again the getter for the `temperatura`: `getTemperatura`